### PR TITLE
Small fixes to README.md

### PR DESCRIPTION
**Pull request by ersen0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@
 ## Download
 Via F-Droid: [com.simplytranslate_mobile](https://f-droid.org/en/packages/com.simplytranslate_mobile/).
 
-For the most up-to-date versions. Download the most recent **Artifact** from the [Actions](https://github.com/ManeraKai/simplytranslate_mobile/actions) page. They may be slow as they're in debug-mode. You should be logged in before you're able to download.
+For the most up-to-date versions, download the most recent **Artifact** from the [Actions](https://github.com/ManeraKai/simplytranslate_mobile/actions) page. They may be slow as they're in debug-mode. You should be logged in before you're able to download.
```
Comment (ManeraKai) [inline README.md:29]: Here it's a one sentence, `For` is like `if`here.

Comment (ersen0) [reply README.md:29]: I understood it as `Download the most recent artifacts for the most up-to-date versions` or `Download the most recent artifacts if you want the most up-to-date versions`. I can revert this part if you want.

Comment (ManeraKai) [reply README.md:29]: Yes, please.

Comment (ersen0) [reply README.md]: Ok, reverted this change.

**Comment by ManeraKai** [comment]
Merged 👍

**Pull request merged**